tests/console/docker_image.pm

```diff
@@ -20,15 +20,15 @@ use version_utils "is_sle";
 sub run {
     select_console "root-console";
 
-    my @image_names = ();
+    my @image_names  = ();
```

Comment:
You define array which always has 1 element. Why not use variable `my $image_name` and `$stable_name`?

Reply:
@dzedro @kravciak there will be more images in the future for sure. That's why I kept it in here.